src/packageManagerBash.py

```diff
@@ -1,21 +1,24 @@
 
 from packageManager import packageManager
 import logging
-import stdplus
+import subprocess
 
 class packageManagerBash(packageManager):
     def isInstalled(self, packageName, package, context, platformConfig):
         if 'isInstalled' in platformConfig:
             command = platformConfig['isInstalled']
-            result = stdplus.run(command, throwOnNonZero = False)
-            return result is 0
+            print(command)
+            result = subprocess.run(command.split())
+            return result.returncode is 0
         pass
 
     def installPackage(self, packageName, package, context, platformConfig):
         if self.isInstalled(packageName, package, context, platformConfig):
-            logging.debug("package '{}' is already installed".format(packageName))
+            logging.debug("package {} is already installed".format(packageName))
             return
-        logging.debug("{}: installing package '{}'".format(self.__class__.__name__, packageName))
+        logging.debug("{}: installing package {}".format(self.__class__.__name__, packageName))
         command = platformConfig['installCommand']
         if command:
-            stdplus.run(command)
+            result = subprocess.run(command.split())
```

Comment:
stdplus.run, with only a command line, would throw an exception on failure. Instead of checking the return code, you could use `check=True` as a parameter to subprocess.run.